### ISSUE TITLE
Polish letters support

### ISSUE THREAD

**Issue by rlewicki** [amaurymartiny/shoot-i-smoke#239]
Platform: iOS and Android
Version: 1.3.1 (iOS), 1.4.0 (Android)
Description:
On both platforms polish letters are not supported by font used  to display our current location (see attached screenshot).
![polish_letters_ios](https://user-images.githubusercontent.com/10658394/66081118-b30ee200-e567-11e9-8471-ed0fc8c2c15f.png)

**Comment by amaurymartiny** [comment]
Yes, you are totally right.

/cc @marceloscoelho, if you have a good idea of an alternative font for here.

**Comment by marceloscoelho** [comment]
Having the font-system as a backup would be good for that and other cases
(arabic, chinese, etc)!

I'm not sure if that's possible with the app, but if we could have
something like:
*font-family: Gotham, sans-serif;*


*Marcelo S. Coelho**Digital Product Designer*

*marcelocoelho.cc <http://marcelocoelho.cc>*
phone: (+33) 07 69 12 14 49
skype: marcelo.scharlau.coelho

Behance <https://www.behance.net/marceloscoelho>  |  Medium
<https://medium.com/@marceloscoelho/>  |  LinkedIn
<https://www.linkedin.com/in/marceloscoelho/>


Em qui, 3 de out de 2019 às 14:00, Amaury Martiny <notifications@github.com>
escreveu:

**Comment by rlewicki** [comment]
Yeah, having fallback font sounds totally reasonable but in this case you need to support all characters at once, detecting if currenly displayed text is supported by font sounds undoable.

What do you think about using one of these: https://www.google.com/get/noto/ ?

**Comment by marceloscoelho** [comment]
Let me mockup with Noto to see if it changes that much, or see if there are
other Google Fonts with big character support (Raleway looks quite close to
the one we are using).


*Marcelo S. Coelho**Digital Product Designer*

*marcelocoelho.cc <http://marcelocoelho.cc>*
phone: (+33) 07 69 12 14 49
skype: marcelo.scharlau.coelho

Behance <https://www.behance.net/marceloscoelho>  |  Medium
<https://medium.com/@marceloscoelho/>  |  LinkedIn
<https://www.linkedin.com/in/marceloscoelho/>


Em qui, 3 de out de 2019 às 18:32, Robert Lewicki <notifications@github.com>
escreveu:
